rest-client/swagger.json

```diff
@@ -162,6 +162,21 @@
       "in": "query",
       "type": "string"
     },
+    "pageSize": {
+      "name": "pageSize",
+      "description": "maximum number of records in each returned page",
+      "in": "query",
+      "type": "integer",
+      "format": "int64",
+      "required": true
```

Comment:
Not required (there is also a default value in the *.yml documentation, so set it there and make sure it's propagated to here.